Core/gb.h

```diff
@@ -44,6 +46,11 @@
 #error Unable to detect endianess
 #endif
 
+#ifdef GB_BIG_ENDIAN
+#define GB_REGISTER_ORDER(a,b,c,d,e,f,h,l) a,f,b,c,d,e,h,l
```

Comment:
This macro would be clearer without arguments (It's rather confusing this way)

Reply:
I changed it, I think this is better?